Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM node
+
+COPY . /rn_website
+WORKDIR /rn_website/website
+
+RUN npm install
```

Comment:
It is possible that `npm` command won't be working correctly after the introduction of Yarn workspaces in the repository. Can you test this out against current `master`?